raft/src/test/java/org/apache/kafka/raft/KafkaRaftClientTest.java

```diff
@@ -278,13 +265,13 @@ public void testEndQuorumStartsNewElectionAfterBackoffIfReceivedFromVotedCandida
             .withVotedCandidate(epoch, otherNodeId)
```

Comment:
The pattern I had in mind was a little different. I was thinking something like this:

```java
        int localId = 0;
        int otherNodeId = 1;
        int epoch = 2;
        Set<Integer> voters = Utils.mkSet(localId, otherNodeId);

        RaftClientTestContext context = new RaftClientTestContext.Builder(localId, voters)
          .withVotedCandidate(epoch, otherNodeId)
          .build()
```

Then we don't have the awkwardness of the partial reliance on the static `LOCAL_ID`. I like this better because the ids have to be explicitly declared in each test case, which makes it easier to follow.

Reply:
Makes sense. I implemented this suggestion.